package.json

```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "start": "node index.js",
-    "test": "NODE_ENV=test mocha --timeout 10000 --require @babel/register 'test/**/**Spec.js'"
+    "test": "env NODE_ENV=test mocha --timeout 10000 --require @babel/register 'test/**/**Spec.js'"
```

Comment:
Mmm. The more reliable way of invoking `mocha` is by its full path, namely `./node_modules/mocha/bin/mocha`. `env` should really have no effect (it's used in shebang lines because they have to give the full path to a command and the locations of many scripting language interpreters is not consistent across systems, which is not applicable here; the binary has to be in `$PATH` for `env` to find it, and if it's in `$PATH` the old version of this line should work). If you want to support globally-installed `mocha`, you can write something like `if test -x ./node_modules/mocha/bin/mocha; then ./node_modules/mocha/bin/mocha [parameters]; else mocha [parameters]` (or some such) but that seems unnecessary.

(Unless the Windows environment simply doesn't support defining variables for a single command, which I really doubt.)

Reply:
Well, when we tried to run on Al's computer, which has Windows 10, only adding the env seemed to work.

Reply:
I think I figured out why: As suggested by the answers to [this not-directly-related question](https://stackoverflow.com/q/9249830), `npm` uses the default shell, which on Windows (nowadays) is PowerShell, and in PowerShell the syntax for running a command with a variable set to some value is different. In which case I'll approve the use of `env`, but repeat my suggestion to invoke `mocha` by its full path; I haven't tested this PR, but I strongly suspect it would result in `Command not found` on my Mac, because I don't have Mocha globally installed and `./node_modules/mocha/bin` isn't in my `$PATH`.

Reply:
Got it. Will do!

Reply:
Got it. Will do!